### DISCUSSION TITLE
沉思錄是什麼

### DISCUSSION THREAD

**Discussion by zhilin93517** [1101-GS4719-C/Python-Community-GS4719-C#2164] [General]
沉思錄在哲學中佔有很重要的地位，在沉思錄之前，哲學是神學。在沉思錄之後，哲學進入理性時代。
從頭質疑所有的一切，尋找不受懷疑的知識基礎，笛卡兒提出了兩個問題: 確定性的問題 Problem of Certainty 和 知識來源的問題 Knowledge Source Problem
笛卡爾認為年輕時接受的很多信念都是假的，在當時建構的基礎所延伸出的每一件事都值得懷疑。所以他決定建立一個全新，穩固且永恆的架構。對於現有架構，不用對每一件事證明其真偽，只須抨擊最根本的基礎，就可以導致整個結構的崩塌。

**Comment by zhilin93517** [comment]
更詳細點說明，在第一卷的主要目的是提出兩個問題，在主旨中有提及，而第二卷針對這兩個問題給出了答案。笛卡爾利用清晰性與明白性的測試去證明上帝的存在，但卻又使用上帝的存在(認為上帝不會欺騙人)去證明清晰與明白可以用於判定命題的確定性。
這樣的推論是循環論證，使用不確定的命題證明另一個命題，再反過來證明原命題，成為循環論證謬誤。

但為了脫離感官影響，笛卡爾認為理性至上，所有的知識可以透過不停的檢驗、推敲，不停地思考來確認任何命題的可靠性，透過不停地推論可以建構出嚴謹而可靠的科學大廈。這樣的知識是來自於理解、理性，而不是透過感官。

**Comment by Chattenoire** [comment]
沉思錄值得看的地方只有第一到第三沉思，其餘就如文中所講，受到宗教觀影響過大，造成循環論證的謬誤。
不過，超越者作為宇宙的第一因仍然是一個所有學科需要面對的終極問題，而這也是神學、哲學在如今還是如此重要的原因之一。
個人認為中世紀哲學還是太神學了，對宗教有興趣再去研讀吧。
